packages/angular/src/generators/application/application.spec.ts

```diff
@@ -69,119 +85,110 @@ describe('app', () => {
-      const tree = await runSchematic('app', { name: 'myApp' }, appTree);
+      await generateApp(appTree);
 
-      expect(tree.readContent('apps/my-app/jest.config.js')).toContain(
+      expect(appTree.read('apps/my-app/jest.config.js', 'utf-8')).toContain(
         `'jest-preset-angular/build/serializers/no-ng-attributes'`
       );
-      expect(tree.readContent('apps/my-app/jest.config.js')).toContain(
+      expect(appTree.read('apps/my-app/jest.config.js', 'utf-8')).toContain(
         `'jest-preset-angular/build/serializers/ng-snapshot'`
       );
-      expect(tree.readContent('apps/my-app/jest.config.js')).toContain(
+      expect(appTree.read('apps/my-app/jest.config.js', 'utf-8')).toContain(
         `'jest-preset-angular/build/serializers/html-comment'`
       );
     });
 
     it('should default the prefix to npmScope', async () => {
-      const noPrefix = await runSchematic(
-        'app',
-        { name: 'myApp', e2eTestRunner: 'protractor' },
-        appTree
-      );
-
-      const withPrefix = await runSchematic(
-        'app',
-        {
-          name: 'myAppWithPrefix',
-          prefix: 'custom',
-          e2eTestRunner: 'protractor',
-        },
-        appTree
-      );
-
       // Testing without prefix
+      await generateApp(appTree, 'myApp', {
+        e2eTestRunner: E2eTestRunner.Protractor,
+      });
 
-      let appE2eSpec = noPrefix
-        .read('apps/my-app-e2e/src/app.e2e-spec.ts')
-        .toString();
-      let workspaceJson = parseJson(noPrefix.read('workspace.json').toString());
+      let appE2eSpec = appTree.read(
+        'apps/my-app-e2e/src/app.e2e-spec.ts',
+        'utf-8'
+      );
+      let workspaceJson = parseJson(appTree.read('workspace.json', 'utf-8'));
       let myAppPrefix = workspaceJson.projects['my-app'].prefix;
 
       expect(myAppPrefix).toEqual('proj');
       expect(appE2eSpec).toContain('Welcome to my-app!');
 
       // Testing WITH prefix
+      await generateApp(appTree, 'myAppWithPrefix', {
+        prefix: 'custom',
+        e2eTestRunner: E2eTestRunner.Protractor,
+      });
 
-      appE2eSpec = withPrefix
-        .read('apps/my-app-e2e/src/app.e2e-spec.ts')
-        .toString();
-      workspaceJson = parseJson(withPrefix.read('workspace.json').toString());
+      appE2eSpec = appTree.read('apps/my-app-e2e/src/app.e2e-spec.ts', 'utf-8');
+      workspaceJson = parseJson(appTree.read('workspace.json', 'utf-8'));
       myAppPrefix = workspaceJson.projects['my-app-with-prefix'].prefix;
 
       expect(myAppPrefix).toEqual('custom');
       expect(appE2eSpec).toContain('Welcome to my-app!');
     });
 
-    // TODO: this test should be fixed
+    // TODO: this should work
+    // This has been carried over from the Angular Devkit Schematic
+    // It seems like Jest is failing as it's trying to look for the
+    // tsconfig in the incorrect place
     xit('should work if the new project root is changed', async () => {
```

Comment:
Can't we fix this now?

Reply:
It seems to be a pretty complex issue from my investigations. I'm pretty sure it was the result of an internal Jest thing. I can maybe look at it again but after I was sinking more time into it than I'd have liked I kept it as it was and updated the comment

Reply:
I was going down a rabbit hole again with this and getting nowhere. Right now, I'm going to leave it skipped and I've marked it down to circle back to it once I have some time